src/Npgsql/Replication/ReplicationConnection.cs

```diff
@@ -527,17 +546,13 @@ async Task SendFeedback(bool waitOnSemaphore = false, bool requestReply = false,
 
             try
             {
-                // Disable the timers while we are sending
-                _sendFeedbackTimer.Change(Timeout.InfiniteTimeSpan, Timeout.InfiniteTimeSpan);
-                _requestFeedbackTimer.Change(Timeout.InfiniteTimeSpan, Timeout.InfiniteTimeSpan);
-
                 var connector = _npgsqlConnection.Connector!;
                 var buf = connector.WriteBuffer;
 
                 const int len = 39;
 
                 if (buf.WriteSpaceLeft < len)
-                    await connector.Flush(true, cancellationToken);
+                    await connector.Flush(async: true, cancellationToken);
```

Comment:
Was this change intentional. If yes, why?

Reply:
Yeah, intentional but not important - I now tend to use explicit parameter names everywhere where the literal isn't self-explanatory (it's hard to know if you're a new-comer what the `true` means alone).